utils/forward-merge.js

```diff
@@ -92,7 +92,9 @@ async function main() {
   let hasConflicts = false;
   let hasUnresolvedConflicts = false;
   const {GITHUB_REF: currentBranch = defaultBranch} = process.env;
-  const [branch, nextBranch] = getBranches(currentBranch.replace('refs/heads/', ''));
+  const [branch, nextBranch] = getBranches(
+    currentBranch.replace('refs/heads/', '').replace('\n', '')
+  );
```

Comment:
This change makes it possible to test locally. The CI doesn't seem to add a newline at the end of the branch while locally it does.